README.md

```diff
@@ -138,6 +152,18 @@ The ramifications of changing the `useHandlebars` setting are explained in the s
 
 [Adding banners](#adding-banners) describes the effects of setting `generateBanners` to `true`.
 
+### Define global constants at compile time using
+
+If you want to provide values for different types of builds (`NODE_ENV` is a popular example), you can define them inside the `dev` and `prod` properties of the `DefinePlugin` section.
+The plugin does a direct text replacement, so the value given to it must include actual quotes inside of the string. You can use alternating quotes, like `"'production'"`, or  use `JSON.stringify('production')`.
+You may take a look at the official [Webpack DefinePlugin docs](https://webpack.js.org/plugins/define-plugin/).
+
+### Automatically load modules instead of requiring / importing them
+
+The `ProvidePlugin` section is an object where the value equals to the module name and the key represents the property name of the window object the module gets mapped to.
+As you can see in the example config above, for jQuery there is the need to define it twice, because bootstrap checks the existence of jQuery via `window.jQuery`.
```

Comment:
> As you can see in the example config above, for jQuery there is the need to define it twice, because bootstrap checks the existence of jQuery via `window.jQuery`.

I would delete that sentence. Too much information for that already bloated readme.

Reply:
Alright 👌